src/web/mjs/connectors/NineEkor.mjs

```diff
@@ -31,20 +31,8 @@ export default class NineEkor extends Connector {
     }
 
     async _getChapters(manga) {
-        let chapterList = [];
-        let request = new Request(new URL(manga.id, this.url), this.requestOptions);
-        let data = await this.fetchDOM(request, 'div.pagination > a.last');
-        let pageCount = data.length === 0 ? 1 : parseInt(data[0].href.match(/page\/(\d+)\//)[1]);
-        for(let page = 1; page <= pageCount; page++) {
-            let chapters = await this._getChaptersFromPage(manga, page);
-            chapterList.push(...chapters);
-        }
-        return chapterList;
-    }
-
-    async _getChaptersFromPage(manga, page) {
```

Comment:
Seems they really dropped chapter pagination, but now tons of chapters are missing on the website (e.g. https://9ekor.com/berserk/).
Guess it cannot be helped, so removing pagination is probably the correct decision.